Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/ExposureDetectionConfig.kt

```diff
@@ -3,9 +3,13 @@ package de.rki.coronawarnapp.appconfig
 import com.google.android.gms.nearby.exposurenotification.ExposureConfiguration
 import de.rki.coronawarnapp.appconfig.mapping.ConfigMapper
 import de.rki.coronawarnapp.server.protocols.internal.ExposureDetectionParameters
+import org.joda.time.Duration
 
 interface ExposureDetectionConfig {
 
+    val maxExposureDetectionsPerDay: Int
```

Comment:
Might be worth renaming to `maxExposureDetectionsPerUTCDay` for future generations 😉